src/main/java/org/springframework/data/tarantool/exceptions/TarantoolEntityMappingException.java

```diff
@@ -0,0 +1,19 @@
+package org.springframework.data.tarantool.exceptions;
+
+import io.tarantool.driver.exceptions.TarantoolException;
+
+/**
+ * Types mapping exception class.
+ *
+ * @author Vladimir Rogach
+ */
+public class TarantoolEntityMappingException extends TarantoolException {
+    public TarantoolEntityMappingException(Class<?> entityCls, String details) {
+        super("Error on operations with entity " + entityCls.getTypeName() + ": " + details);
```

Comment:
Not a comprehensive message for the user. We need to reformulate it to something like `Failed to convert the entity %s to ...`

Reply:
Renamed class and refactored usage.